pkg/client/unversioned/persistentvolumeclaim.go

```diff
@@ -25,6 +25,10 @@ import (
 	"k8s.io/kubernetes/pkg/watch"
 )
 
+const (
+	persistentVolumeClaimResourceName string = "persistentVolumeClaims"
```

Comment:
this doesn't match what was in the old code.  Are the names of the API endpoints incorrect?